TableTopCampaignManager/DiscordBot.cs

```diff
@@ -49,5 +52,47 @@ public async Task InitializeCommands()
                 Console.WriteLine(json);
             }
         }
+        private async Task SlashCommandHandler(SocketSlashCommand command)
+        {
+            switch (command.Data.Name)
+            {
+                case "roll":
+                    await HandleRollCommand(command);
+                    break;  
+            }
+        }
+        private static int[] CalculateRoll(string userInput, SocketSlashCommand command)
+        {
+            Random random = new Random();
+            int numberOfRolls = Int32.Parse(userInput.Split('d')[0]);
+            int dieValue = Int32.Parse(userInput.Split('d')[1]);
+            int[] calculatedValues = new int[numberOfRolls];
+            if (numberOfRolls > 10)
+            {
+                command.RespondAsync(text: "That's more than 10 rolls! 10 is the maximum.");
+                return null;
+            }
+            else
+            {
+                for (int i = 0; i < numberOfRolls; i++)
+                {
+                    calculatedValues[i] = random.Next(1, dieValue + 1);
```

Comment:
why dieValue+1?

Reply:
Because when using `random.Next`, it calculates a value that is >= the left value, but only < the right value. Not <=